src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  CodeSnippet,
+  CodeSnippetBlockAppearance,
+  Spinner,
+} from "@canonical/react-components";
+import { useState } from "react";
+
+import {
+  getCrossModelQueryLoaded,
+  getCrossModelQueryLoading,
+  getCrossModelQueryResults,
+} from "store/juju/selectors";
+import { useAppSelector } from "store/store";
+
+enum CodeSnippetView {
+  TREE = "tree",
+  JSON = "json",
+}
+
+const ResultsBlock = (): JSX.Element | null => {
+  const isCrossModelQueryLoading = useAppSelector(getCrossModelQueryLoading);
+  const isCrossModelQueryLoaded = useAppSelector(getCrossModelQueryLoaded);
+  const crossModelQueryResults = useAppSelector(getCrossModelQueryResults);
+  const [codeSnippetView, setCodeSnippetView] = useState<CodeSnippetView>(
+    CodeSnippetView.TREE
+  );
+  const codeSnippetContent = {
+    tree: "",
```

Comment:
This will be done in https://warthogs.atlassian.net/browse/WD-5820 but will require a change to react-components to support a ReactNode instead of a string as the code prop: https://warthogs.atlassian.net/browse/WD-5870.